app/javascript/components/FiltersBar/FiltersBar.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Form, FormGroup, Input, Label } from 'reactstrap';
+
+const SelectInput = (props) => {
```

Comment:
You can use destruction for the actual props here:
`({ label, handleChange, ... })`